docs/source/admin/upgrading.rst

```diff
@@ -5,20 +5,20 @@ Upgrading JupyterHub
 JupyterHub offers easy upgrade pathways between minor versions. This
 document describes how to do these upgrades.
 
-If you are using :ref:`a JupyterHub distribution <index/distributions>`, you
+If you are using `distributions <https://github.com/ToobaJamal/jupyterhub/blob/main/docs/source/index.rst#distributions>`__, you
 should consult the distribution's documentation on how to upgrade. This
-document is if you have set up your own JupyterHub without using a
+document is useful if you have set up your own JupyterHub without using a
 distribution.
 
-It is long because is pretty detailed! Most likely, upgrading
-JupyterHub is painless, quick and with minimal user interruption.
+The steps are discussed in detail. Most likely, 
```

Comment:
This more formal language communicates less information. The gist to get at here is that while the doc is long, that's because it contains far more detail than the reader likely needs, but it's here for reference if anything goes wrong.